packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -4,191 +4,201 @@
-                                data-count="alerts.mitigation.length"
-                                data-when="{
-                                      '0': ('ip_alerts_mitigation' | translate),
-                                      'one': ('ip_alerts_mitigation_1' | translate),
-                                      'other': ('ip_alerts_mitigation_other' | translate: { count: {} })
-                                  }"
-                            >
-                            </span>
-                            <button
-                                class="btn btn-link"
-                                type="button"
-                                data-ng-if="!singleService()"
-                                data-ng-repeat="ip in alerts.mitigation"
-                                data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
-                                data-ng-bind="ip.ip.routedTo.serviceName"
-                            ></button>
-                        </p>
-                    </div>
+                    <p data-ng-if="alerts.spam.length">
+                        <span
+                            class="d-block"
+                            data-ng-pluralize
+                            data-count="alerts.spam.length"
+                            data-when="{
+                                    '0': ('ip_alerts_spam' | translate),
+                                    'one': ('ip_alerts_spam_1' | translate),
+                                    'other': ('ip_alerts_spam_other' | translate: { count: {} })
+                                }"
+                        >
+                        </span>
+                        <button
+                            class="btn btn-link"
+                            type="button"
+                            data-ng-if="!singleService()"
+                            data-ng-repeat="ip in alerts.spam"
+                            data-ng-click="navigateToService(ip.ip.routedTo.serviceName)"
+                            data-ng-bind="ip.ip.routedTo.serviceName"
+                        ></button>
+                    </p>
 
-                    <div
-                        class="alert alert-success"
-                        data-ng-if="!alertsCount()"
-                        role="alert"
-                    >
+                    <p data-ng-if="alerts.antihack.length">
                         <span
+                            class="d-block"
+                            data-ng-pluralize
+                            data-count="alerts.antihack.length"
+                            data-when="{
+                                    '0': ('ip_alerts_antihack' | translate),
+                                    'one': ('ip_alerts_antihack_1' | translate),
+                                    'other': ('ip_alerts_antihack_other' | translate: { count: {} })
+                                }"
+                        >
+                        </span>
```

Comment:
Same as https://github.com/ovh/manager/pull/6229/files#r777884281

Reply:
No modifications done here. It's showing in the changes section because of the change in the layout structure.